docs/Upcoming.md

```diff
@@ -9,7 +9,7 @@ This page lists upcoming changes and features that are going to be documented in
 <section class="blocks alt">
   <a class="bl two" href="/API/core/app-actions.html">
     <div>
-      <span class="pin">Open beta</span>
+      <span class="pin">Now available to all customers</span>
```

Comment:
We can just remove the beta tag without adding this.

```suggestion
```

Reply:
Fixed